massa-node/base_config/config.toml

```diff
@@ -38,6 +38,38 @@
     # whether to enable WS.
     enable_ws = false
 
+[grpc]
+    # whether to enable gRPC
+    enabled = false
+    # whether to add HTTP 1 layer
+    accept_http1 = false
+    # whether to enable gRPC reflection(introspection)
+    enable_reflection = true
+    # bind for the Massa gRPC API
+    bind = "0.0.0.0:33037"
+    # which compression encodings does the server accept for requests
+    accept_compressed = "Gzip"
+    # which compression encodings might the server use for responses
+    send_compressed = "Gzip"
```

Comment:
List the options (or a link to documentation to help users choose).

Reply:
We have Only Gzip for now

Reply:
Ok, then maybe add "Only "Gzip currently supported" to be clearer? :)